StardewArchipelago/GameModifications/CodeInjections/TheaterInjections.cs

```diff
@@ -194,6 +194,10 @@ public static bool MakeMapModifications_JojamartAndTheater_Prefix(Town __instanc
                     {
                         __instance.crackOpenAbandonedJojaMartDoor();
                     }
+                    if (!Game1.player.mailReceived.Contains(string.Join("ap",ABANDONED_JOJA_MART)))

```

Comment:
Why not send this letter when receiving the actual movie theater, instead of here?